packages/docs/src/routes/docs/(qwikcity)/action/index.mdx

```diff
@@ -20,7 +20,7 @@ Actions can be declared using the `routeAction$()` or `globalAction$()` exported
 import { component$ } from '@builder.io/qwik';
 import { routeAction$, Form } from '@builder.io/qwik-city';
 
-export const useAddUser = routeAction$(async (data, requestEvent) => {
+export const useAddUser = routeAction$(async (data) => {
```

Comment:
even if `requestEvent` not used, i would rather leave it there. it's good for Qwik AI to know what to expect in every argument

Reply:
Got it!